src/massive/munit/TestRunner.hx

```diff
@@ -182,29 +195,88 @@ class TestRunner implements IAsyncDelegateObserver
         suiteIndex = 0;
         clientCompleteCount = 0;
         Assert.assertionCount = 0; // don't really like this static but can't see way around it atm. ms 17/12/10
-        testSuites = [for(suiteType in testSuiteClasses) Type.createInstance(suiteType, emptyParams)];
+        emptyParams = new Array();
+        asyncDelegates = new Array<AsyncDelegate>();
+        testSuites = new Array<TestSuite>();
         startTime = Timer.stamp();
 
-        #if (neko || cpp || java)
-		var self = this;
-		var runThread:Thread = Thread.create(function()
-		{
-			self.execute();
-			while (self.running)
-			{
-				Sys.sleep(.2);
-			}
-			var mainThead:Thread = Thread.readMessage(true);
-			mainThead.sendMessage("done");
-		});
-		runThread.sendMessage(Thread.current());
-		Thread.readMessage(true);
+        for (suiteType in testSuiteClasses)
+        {
+            testSuites.push(Type.createInstance(suiteType, new Array()));
+        }
```

Comment:
200, 203-206 This code does the same thing as before, but now it has become more.